src/main.js

```diff
@@ -73,7 +72,6 @@ function changeViewSavedView() {
   viewSavedView.classList.remove('hidden');
   homeView.classList.add('hidden');
   formView.classList.add('hidden');
-  showSavedCovers();
```

Comment:
This was causing the array multiplying bug... Makes so much sense in retrospect.